engine/compiler/compiler.go

```diff
@@ -163,22 +183,35 @@ func (c *Compiler) Compile(ctx context.Context, args runtime.CompilerArgs) runti
 				},
 			),
 		)
+
+		// TODO: Remove when lespec.Step is with Files slice
+		_ = clonepath
+		_ = clonefile
+
 		cmd, args := oshelp.GetCommand(pipelineOS, clonepath)
 		spec.Steps = append(spec.Steps, &engine.Step{
-			Name:      "clone",
+			Step: lespec.Step{
+				ID:   random(),
```

Comment:
Also, set a Key field in all the steps. It can be same as ID.

Reply:
What is the exact name of the field? I can't find it...

Reply:
https://github.com/harness/lite-engine/blob/main/api/api.go#L47

Reply:
LogKey